bitcoin/src/lib.rs

```diff
@@ -880,6 +893,46 @@ impl BitcoinCoreApi for BitcoinCore {
         Ok(())
     }
 
+    async fn rescan_electrs_for_addresses<A: PartialAddress + Send + Sync + 'static>(
+        &self,
+        addresses: Vec<A>,
+    ) -> Result<(), Error> {
+        for address in addresses.into_iter() {
+            let address = address.encode_str(self.network)?;
+            let all_transactions =
+                esplora_btc_api::apis::address_api::get_address_tx_history(&self.electrs_config, &address).await?;
+            let confirmed_payments_to = all_transactions.into_iter().filter(|tx| {
+                if let Some(status) = &tx.status {
```

Comment:
When is this value `None`? Should we return false if doesn't exist?

Reply:
Apparently the `Transaction` type from electrs has most of its fields as an `Options`, not sure why.  I don't see in what case it would be None, but in any case we only import confirmed transactions and those would have `status: { confirmed: true }`. The API seems to always return the status object just fine.